src/models/media.ts

```diff
@@ -0,0 +1,32 @@
+import type { SupportedMediaType } from '~/constants/media'
+
+export interface Tag {
+  name: string
+}
+
+export interface ApiMedia {
```

Comment:
We do transform the data that comes from the API immediately, adding `frontendMediaType`, as well as creator, title and tags if necessary:
https://github.com/WordPress/openverse-frontend/blob/841cf25197e9a857b64cd77a1b809143b9fd1a14/src/utils/decode-media-data.js#L10-L22

So, I think it's best to use a single `Media` type (MediaDetails ? Media is kind of a nebulous term, but I'm not sure what the best alternative would be)

Reply:
Right hmmm...

I wonder if localizing where the `ApiMedia` type exists instead of having it in a shared space? I can just see a contributor being confused about which one to import and use in other functions.

At the very least some doc comments explaining when and where to use one or the other would be helpful for this.

Reply:
Localised `ApiMedia` to `decode-media-data.ts` and added a docsting in 5927df4.